app/controllers/books_controller.rb

```diff
@@ -0,0 +1,7 @@
+class BooksController < ApplicationController
+
+    def show
+      @book = Book.find(params[:id])
+      @review = Review.new
```

Comment:
Since none of the user stories ask for a review to be created you don't need line 5.